scarlet-websocket-okhttp/src/main/java/com/tinder/scarlet/websocket/okhttp/OkHttpClientUtils.kt

```diff
@@ -16,5 +18,16 @@ fun OkHttpClient.newWebSocketFactory(requestFactory: RequestFactory): WebSocket.
 }
 
 fun OkHttpClient.newWebSocketFactory(url: String): WebSocket.Factory {
+    if (url.startsWith("ws://")) {
+        try {
+            if (Build.VERSION.SDK_INT >= 23 &&
+                    !NetworkSecurityPolicy.getInstance().isCleartextTrafficPermitted()) {
+                throw RuntimeException("Android configuration does not permit cleartext traffic.")
```

Comment:
the error type and message in okhttp sound better `UnknownServiceException` with `CLEARTEXT communication to $host not permitted by network security policy`, and

Reply:
Did part of this comment get cut off?